scripts/npm/prepublish/lib/docs.js

```diff
@@ -0,0 +1,46 @@
+var exec = require('child-process-promise').exec;
+var util = require('../../lib');
+
+function docs(version) {
+  checkoutDocs()
```

Comment:
Change this line to `return checkoutDocs()` so that the `docs` function is returning a promise.